app/javascript/lib/validation/validate-file-content.js

```diff
@@ -144,17 +144,72 @@ function validateEqualCount(headers, annotTypes) {
   return issues
 }
 
+/**
+ * Verify "GENE" is present as the first column in the first row for an Expression Matrix file
+ */
+function validateGeneInHeader(firstRowTable) {
+  const issues = []
+
+  if (firstRowTable[0][0] !== 'GENE') {
+    const msg = 'Dense matrices require the first value of the file to be "GENE". ' +
```

Comment:
Technically, there are 2 cases that we allow as per `scp-ingest-pipeline`:

1. First keyword is `GENE` (See [here](https://github.com/broadinstitute/scp-ingest-pipeline/blob/development/ingest/expression_files/dense_ingestor.py#L215))
2. There is no `GENE` keyword, and the first line has exactly 1 entry less than each successive line (also referred to as "R formatted", since this is what the output of `write.table` will do).  See [here](https://github.com/broadinstitute/scp-ingest-pipeline/blob/development/ingest/expression_files/dense_ingestor.py#L129).

Looking at SCP-3766, I see this isn't called out in that ticket, so obviously there was no way for you to know that - but we should maintain feature parity with `scp-ingest-pipeline`, where possible.

Reply:
That being said - the validation does work as expected for the `GENE` header.

Reply:
oo thanks for catching that! I will make that update

Reply:
oh looking at the ingest-pipeline code it also appears "gene" would be acceptable as well - as in it is not case sensitive - is that correct?

Reply:
Yes - you are correct.  Good catch!  At one point, we also accepted an empty string (`""`), but in looking at the code that doesn't appear to be the case anymore.

Reply:
It looks like there is actually a bug currently in the code with the `sniffdelimiter` function such that R-formatted expression matrices are not being handled correctly. Because of that I am thinking I will keep this PR to only allow exp matrices with "gene" headers and make a followup ticket to address the bug in the delimiter and then allow R-formatting as well. Please let me know if that seems reasonable, thanks!